bundle/README.md

```diff
@@ -71,6 +71,36 @@ For a list of available configuration for the `values` field, refer to [Istio's
 - [CNI parameters](https://artifacthub.io/packages/helm/istio-official/cni?modal=values)
 - [ZTunnel parameters](https://artifacthub.io/packages/helm/istio-official/ztunnel?modal=values)
 
+## Installing the istioctl tool
+
+The istioctl tool is a configuration command line utility that allows service operators to debug and diagnose Istio service mesh deployments.
+
+### Prerequisites
+
+Use an istioctl version that is the same version as the Istio control plane for the Service Mesh deployment. See [Istio Releases](https://github.com/istio/istio/releases) for a list of valid releases, including Beta releases.
+
+### Procedure
+
+1. Confirm if you have istioctl installed, and if so which version, by running the following command at the terminal:
+
+  ```sh
+    $ istioctl version
+    ```
+1. Upgrade istioctl by running the following command at the terminal. Replace <version> with a valid release version of Istio: 
+
+  ```sh
+    $ curl -sL https://istio.io/downloadIstioctl | ISTIO_VERSION=<version> sh -
+    ```
```

Comment:
@jwendell what path is being updated with the following syntax? Is it the path for istioctl or something else?

Reply:
`PATH` is a command line thing. It specifies which directories the system should look when running executable files.

For instance, you usually type `ls` instead of the full location of the `ls` binary, which is `/usr/bin/ls`. This is because `/usr/bin` is part of your `PATH`. Type `echo $PATH` to see the full list of directories. They are separated by colon.

What we do below is to add the directory of `istioctl` to the PATH so that we can invoke it by just typing `istioctl` rather than having to type `~/.istiovtl/bin/istioctl` every time.

Reply:
Done